pontoon/projects/views.py

```diff
@@ -81,18 +84,29 @@ def project(request, slug):
 def ajax_teams(request, slug):
     """Project Teams tab."""
     project = get_object_or_404(
-        Project.objects.visible_for(request.user).available(), slug=slug
+        cast(
+            BaseManager[Project], Project.objects.visible_for(request.user).available()
+        ),
+        slug=slug,
```

Comment:
Is this really necessary?

Reply:
The `cast()` is a no-op at runtime, but with it the type of `project` is correct within this function.

Reply:
While I appreciate the intent behind improving type safety, our Python codebase does not use static typing consistently. Introducing it in just this one place creates an inconsistency that might confuse other developers who are not expecting type hints.

Reply:
On the other hand, type hints were how I was able to catch the bug fixed in #3562. Without them, working with Pontoon code is much more difficult and more error-prone, in particular for developers not previously familiar with it.

We should be working towards more type hinting being available in Pontoon.